tdms/CMakeLists.txt

```diff
@@ -102,6 +102,7 @@ set(SOURCES
     src/matrix_collection.cpp
     src/mesh_base.cpp
     src/numerical_derivative.cpp
+    src/openandorder.cpp
```

Comment:
Is it worth us just doing another `CMake` `GLOB` here to catch all the `.cpp` fdiles in the `src/` directory? I can imagine with `hdf5` impending we're going to want to avoid a series of one-line changes to the `CMakeLists.txt` file.

Reply:
Oh. Yeah. Very good point.